File: app/src/androidTest/java/io/github/fvasco/pinpoi/importer/ZipImporterTest.java
package io.github.fvasco.pinpoi.importer;

import org.junit.Test;

import java.util.List;

import io.github.fvasco.pinpoi.model.Placemark;

/**
 * @author Francesco Vasco
 */
public class ZipImporterTest extends AbstractImporterTestCase {

    @Test
    public void testImportImpl() throws Exception {
        final List<Placemark> list = importPlacemark(new ZipImporter(), "test3.kmz");
        assertEquals(3, list.size());
    }
}

File: app/src/androidTest/java/io/github/fvasco/pinpoi/importer/ZipImporterTest.kt
package io.github.fvasco.pinpoi.importer

import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.Assert.assertEquals
import org.junit.Test
import org.junit.runner.RunWith

/**
 * @author Francesco Vasco
 */
@RunWith(AndroidJUnit4::class)
class ZipImporterTest : AbstractImporterTestCase() {

    @Test
    @Throws(Exception::class)
    fun testImportImpl() {
        val list = importPlacemark(ZipImporter(), "test3.kmz")
        assertEquals(3, list.size)
    }
}